### PR TITLE
Fix typo that invalid options in argparse

### PR DESCRIPTION
**Pull request by ogaclejapan**
## Description

argparse command is very cool!! 
I fixed it because I found a typo in the document.

```
> echo "argparse --name=my_function 'h/help' 'n/name:' -- $argv" > test.fish
> fish test.fish
my_function: Invalid option spec 'n/name:' at char ':'
```

### CODE REVIEW

File: doc_src/argparse.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ The following `argparse` options are available. They must appear before all OPTI
 Using this command involves passing two sets of arguments separated by `--`. The first set consists of one or more option specifications (`OPTION_SPEC` above) and options that modify the behavior of `argparse`. These must be listed before the `--` argument. The second set are the arguments to be parsed in accordance with the option specifications. They occur after the `--` argument and can be empty. More about this below but here is a simple example that might be used in a function named `my_function`:
 
 \fish
-argparse --name=my_function 'h/help' 'n/name:' -- $argv
+argparse --name=my_function 'h/help' 'n/name' -- $argv
```
Comment (zanchey) [inline doc_src/argparse.txt]: I think this was leftover from an early version of the argparse syntax. The correct change here is `n/name=` instead - do you want to redo the commit?

Comment (ogaclejapan) [reply doc_src/argparse.txt]: Thank you for your review.
Yes, I will fix it soon 👍

**Comment by ogaclejapan** [comment]
I fixed it. 👍

**Comment by zanchey** [comment]
Awesome, thanks!

**Pull request merged**